streamingpro-spark-2.4.0-adaptor/src/main/java/org/apache/spark/sql/jdbc/UpsertUtils.scala

```diff
@@ -231,3 +233,50 @@ object MysqlUpsertBuilder extends UpsertBuilder with Logging {
   }
 }
 
+/**
+ * Oracle has Merge INTO statement since oracle 9i
+ * The following Merge statement updates employee.address column if id column equals to given value
+ * inserts a record if not.
+  MERGE INTO employees
```

Comment:
When we concat the table name and field name of a sql statement, should we add some escape characters, such as:

````
MERGE INTO `table1`
````

Also, if the table name or field name comes with an escape character, we shouldn't add it again.
@chncaesar @allwefantasy

Reply:
` is not allowed to quote an identifier. See  error below:
![image](https://user-images.githubusercontent.com/12869649/160036013-06b3189a-1c1b-4708-903c-ec3cedca26af.png)

Double quote " is not recommened as well. Because double-quote makes identifiers case sensitive. See example:
![image](https://user-images.githubusercontent.com/12869649/160036305-ad4ddade-2d38-43c0-9aee-052c3dc04ec1.png)

This is because table/column names are stored in upper-case if not double-quotted;  and stored as it as if double-quotted.

Reply:
So does this means oracle do not support  quote like "`" ？